admin-ui/app/utils/RouteLoader.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { Suspense, lazy, ComponentType } from 'react'
+import GluuSuspenseLoader from 'Routes/Apps/Gluu/GluuSuspenseLoader'
+
+// Route loading utility
+export const createLazyRoute = (importFn: () => Promise<{ default: ComponentType<any> }>) => {
+  const LazyComponent = lazy(importFn)
+
+  return (props: any) => (
+    <Suspense fallback={<GluuSuspenseLoader />}>
+      <LazyComponent {...props} />
+    </Suspense>
+  )
+}
```

Comment:
Expose preload to make preloadRoute work.

createLazyRoute returns a function component without a preload method, so preloadRoute never does anything. Attach preload = importFn.

```diff
 export const createLazyRoute = (importFn: () => Promise<{ default: ComponentType<any> }>) => {
   const LazyComponent = lazy(importFn)
 
-  return (props: any) => (
-    <Suspense fallback={<GluuSuspenseLoader />}>
-      <LazyComponent {...props} />
-    </Suspense>
-  )
+  const Wrapper: any = (props: any) => (
+    <Suspense fallback={<GluuSuspenseLoader />}>
+      <LazyComponent {...props} />
+    </Suspense>
+  )
+  // Allow preloading the chunk ahead of time
+  Wrapper.preload = importFn
+  return Wrapper
 }
```